src/scancode/api.py

```diff
@@ -181,15 +181,25 @@ def get_licenses(location, min_score=0,
         qspans.append(match.qspan)
 
         detected_expressions.append(match.rule.license_expression)
-
-        detected_licenses.extend(
-            _licenses_data_from_match(
+        if "unknown" in match.rule.license_expression:
```

Comment:
matches will have a `is_unknown` flag you added :P Use that instead of checking the license_eexpression.

Reply:
Oh yeah! Actually that PR is not yet merged therefore changes were not reflected in this branch which was giving errors, that's why I have used this .